src/status_im/contexts/wallet/tokens/events.cljs

```diff
@@ -116,7 +116,7 @@
                                {}
                                raw-data)]
      {:db (-> db
-              (assoc-in [:wallet :tokens :market-values-per-token] market-values)
+              (update-in [:wallet :tokens :market-values-per-token] merge market-values)
```

Comment:
Before we requested market values just one time so `assoc-in` was used, but now we do this more often with smaller subset of tokens, so need an `update-in`